client/tinychain/ml/mlr.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
TinyChain requires Python >= 3.8 so you don't need this line

more info: https://stackoverflow.com/questions/25205395/what-is-the-default-encoding-method-for-code-assumed-by-python-interpreter